jazz_environments/components/utils.js

```diff
@@ -65,7 +65,7 @@ var getSchemaKeyName = function(key) {
 
 // convert object returned from the database, as per schema
 var formatEnvironment = function(environment, format) {
-    if (environment === undefined || environment === null) {
+    if (environment === undefined || !environment) {
```

Comment:
don't need a check for undefined since you are already checking for `!environment`

Reply:
Done